worker/send_new_user_email.js

```diff
@@ -7,7 +7,7 @@ module.exports = function(notifier_messager, mailroom) {
       return process.nextTick(cb);
     }
 
-    var mailTemplate = event.data.teach ? "user_created_teach" : "user_created";
+    var mailTemplate = event.data.client_id === teach_client_id ? "user_created_teach" : "user_created";
```

Comment:
I just added a check above that sets it to false if it's a falsey value to handle this edge case